{{ cookiecutter.package_name|slugify }}/.flake8

```
@@ -7,5 +7,5 @@ max_line_length = 100
 max_complexity = 10
 docstring_style = numpy
 doctests = True
-ignore = E203,E501,FS003,S101,W503
+ignore = E203,E501,FS003,S101,W503,DAR103
```

Could you put this one up front (so the list is alphabetically sorted)?
```suggestion
ignore = DAR103,E203,E501,FS003,S101,W503
```

---

Done